modules/core/src/v2/coins/cspr.ts

```diff
@@ -201,4 +222,63 @@ export class Cspr extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Explain a Casper transaction from Raw Tx
+   *
+   * @param params
```

Comment:
explain here list of params

Reply:
You're right, I can improve this jsdoc. Thanks!

Reply:
Done!